geode-dunit/src/distributedTest/java/org/apache/geode/test/dunit/tests/AsyncInvocationTimeoutDistributedTest.java

```diff
@@ -40,12 +41,20 @@
 
   private static final long TIMEOUT_MILLIS = getTimeout().toMillis();
 
-  private static final AtomicReference<Long> threadId = new AtomicReference<>();
+  private static final AtomicReference<Long> threadId = new AtomicReference<>(0L);
   private static final AtomicReference<CountDownLatch> latch = new AtomicReference<>();
 
   @Rule
   public DistributedRule distributedRule = new DistributedRule();
 
+  @Before
+  public void setUp() {
+    getVM(0).invoke(() -> {
+      latch.set(null);
```

Comment:
Is it better to initialize `latch` to `CountDownLatch(0)` instead of `null`? Also initialize it in line 45, when `latch` is declared?

Reply:
Yes, it would be better to avoid nulls and use CountDownLatch(0). Thanks!

Reply:
I think I've resolved this. Please take a look. Thanks!